sources/updater/updog/src/main.rs

```diff
@@ -31,6 +31,7 @@ const TARGET_ARCH: &str = "aarch64";
 
 const TRUSTED_ROOT_PATH: &str = "/usr/share/updog/root.json";
 const MIGRATION_PATH: &str = "/var/lib/bottlerocket-migrations";
+const METADATA_PATH: &str = "/var/cache/bottlerocket-metadata";
```

Comment:
That would be an extra layer of defense, but we don't strictly need it since our root of trust is immutable.

I'd rather approach that as a different issue since it involves giving `updog` a label of its own (`update_t`) and allowing processes to transition to that label. That'll be much more satisfying once the update API is in place; otherwise, we also have to let the container process labels use that transition, or the operator won't work correctly.